docs/Using-the-dataset-validator.md

```diff
@@ -414,3 +422,78 @@ As an example, the command for the "Example 1" listed above incorporating the `-
 ```
 ./validateData.py -s ../../../test/scripts/test_data/study_es_0/ -P ../../../../../src/main/resources/portal.properties -u http://localhost:8080/cbioportal -v
 ```
+
+## Running the validator for multiple studies
+The importer folder `<your_cbioportal_dir>/core/src/main/scripts/importer` also contains a script for running the validator for multiple studies:
+```bash
+./validateStudies.py --help
+```
+The following parameters can be used:
+```console
+usage: validateStudies.py [-h] [-d ROOT_DIRECTORY] [-l LIST_OF_STUDIES]
+                          [-html HTML_FOLDER]
+                          [-u URL_SERVER | -p PORTAL_INFO_DIR | -n]
+                          [-P PORTAL_PROPERTIES]
+
+Wrapper where cBioPortal study validator is run for multiple studies
+
+optional arguments:
+-h, --help            show this help message and exit
+-d ROOT_DIRECTORY, --root-directory ROOT_DIRECTORY
+                      Path to directory with all studies that should be
+                      validated
+-l LIST_OF_STUDIES, --list-of-studies LIST_OF_STUDIES
+                      List with paths of studies which should be validated
+-html HTML_FOLDER, --html-folder HTML_FOLDER
+                      Path to folder for output HTML reports
+-u URL_SERVER, --url_server URL_SERVER
+                      URL to cBioPortal server. You can set this if your URL
+                      is not http://localhost/cbioportal
```

Comment:
Found 2 more!

Could you update these as well with our http://localhost:8080/cbioportal update?